app/src/main/java/com/springsamurais/toyswap/ui/addlisting/AddListingClickHandlers.kt

```diff
@@ -0,0 +1,26 @@
+package com.springsamurais.toyswap.ui.addlisting
+
+import android.content.Context
+import android.content.Intent
+import android.view.View
+import com.springsamurais.toyswap.ui.mainactivity.MainActivity
+
+class AddListingClickHandlers(private var context: Context) {
+
+    fun captureImageButtonClicked(view: View) {
+        // Moazam has code for opening camera
+    }
+
+    fun accessGalleryButtonClicked(view: View) {
+        // Moazam has code for accessing gallery
```

Comment:
Before merge, @mmoazam could potentially add the open gallery logic if it's ready to go.